www/front_src/src/App/index.tsx

```diff
@@ -34,7 +34,7 @@ const useStyles = makeStyles((theme) => ({
   },
   mainContent: {
     backgroundColor: theme.palette.background.paper,
-    height: '100%',
+    height: `calc(100vh - ${theme.spacing(9)})`,
```

Comment:
to what does theme.spacing(9) correspond? if it's the height of some element, we should declare a variable with the value in order to reuse it accross both of them (the targeted element and the mainContent)

Reply:
The 100% value works too

Reply:
@Thebarda  le probléme est dans les pages Smarty, theme.spacing(9) est la taille de header  + un margin, ouais je vais faire ça dans une variable

Reply:
![Screenshot 2022-08-31 at 10 09 50](https://user-images.githubusercontent.com/12515407/187627350-c8b3bf0a-15ff-4326-b8c1-206073773a5c.png)
Uuuh it works for React pages it doesn't for Smarty pages

Reply:
The correct spacing is 8. Using 9, it displays a small white band on dark mode

Reply:
Fixed